inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/consume/BaseInlongConsume.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.pojo.consume;
+
+import io.swagger.annotations.ApiModel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * The base parameter class of InlongGroup, support user extend their own business params.
+ */
+@Data
+@AllArgsConstructor
+@ApiModel("Base info of inlong group")
+public class BaseInlongConsume {
+
+    // you can add extend parameters in this class
+
+}
```

Comment:
Suggest adding one blank line after the end of `}`, then it will not appear this:

<img width="115" alt="image" src="https://user-images.githubusercontent.com/31994335/200123281-0c60a7de-6215-477d-be2e-29ca2509f801.png">